spec/integration/fetching_linked_items_spec.rb

```diff
@@ -1,12 +1,13 @@
 require 'rails_helper'
+include ActiveSupport::Testing::TimeHelpers
 
 describe "Fetching linked items", type: :request do
   describe "GET /incoming-links/:base_path_without_root" do
     it "returns the items linked to an item" do
-      Timecop.freeze do
+      travel_to Time.now do
```

Comment:
Did you try calling `Timecop.freeze` in a `before` block? I suspect that might work (although it doesn't really matter, since you've got it working another way)